README.md

```diff
@@ -1,56 +1,75 @@
+
+## This app is for you if you want to...
+
+-   📈 track their net worth
+-   💸 track your expenses to better understand where they are spending their money
+-   👀 keep track of your investments
+-   🪦 keep everything in one place like Mr. Rip
 
 What features do we want to implement?
--   Track expenses (even recurrent ones)
--   Track different bank accounts
--   Graphs, statistics and reports
--   All data are stored locally (with the option to share it in different platforms)
--   Track investments: Stocks, Bonds, Crypto, Private equity
--   Track taxes
--   Net worth on different currencies
--   Import/export data
--   Link bank accounts that support the OpenBanking PSD2 API
--   Advanced onboarding
-
-#### Phase 1
-We will not build everything at once so we will start from the basic features and then we will move to the others.
-Phase 1 will be completed when the following features will ready:
--   Expenses (even recurrent ones) tracker
--   Income tracker
--   Bank accounts balance (without the use of API)
--   Basic stats
--   Data stored only locally
+-   💸 Track expenses (even recurrent ones)
+-   🏦 Track different bank accounts
+-   📊 Graphs, statistics and reports
+-   💻 All data are stored locally (with the option to share it in different platforms)
+-   👀 Track investments: Stocks, Bonds, Crypto, Private equity
+-   💶 Track taxes
+-   💴 Net worth on different currencies
+-   ↔️ Import/export data
+-   🔗 Link bank accounts that support the OpenBanking PSD2 API
+-   🤓 Advanced onboarding
+
+<div align="center">
+<img src="./assets/Sossoldi-portfolio-card-image.png" width="500" alt="Sossoldi icon">
+</div>
+
+## Phase 1 (Ongoing)
+We will not build the complete app all at once. We will start from the basic features and then we will move onto the others.
+Phase 1 will be completed when the following features will be ready:
+-   💸 Expenses (even recurrent ones) tracker
+-   🤑 Income tracker
+-   ⚖️ Bank accounts balance (without the use of API)
+-   📊 Basic stats
+-   📱 Data stored only locally
 
 The pages that will be implemented are:
 -   Dashboard
 -   Movements page
 -   Basic settings
 -   Basic onboarding
 
-#### How you can help
+## Contributing
+
+If you want to help with this project you are more than welcome! Sossoldi is completely free and open source, the best place to start is from the issue tab and look for something that seems interesting to you. If are unsure on where to start feel free to reach out to us on [Discord](https://discord.com/channels/919139369774891088/1013368203575038012), we will be more than happy to help you find what's best for you!
 
-If you want to help with this project you are more than welcome!
-Here is a series of link to get started:
+### Useful links to get started
```

Comment:
I would add on the links section also to the UI/UX case study: https://www.federicopozzato.it/portfolio/sossoldi-personal-finance-app

Reply:
Done! 👍🏻